worlds/sc2/mission_groups.py

```diff
@@ -192,3 +192,10 @@ def get_all_group_names(cls) -> Set[str]:
     SC2Mission.THE_HOST.mission_name,
     SC2Mission.SALVATION.mission_name,
 ]
+
+for mission in SC2Mission:
+    if mission.flags & MissionFlag.HasRaceSwap:
+        short_name = mission.mission_name[:mission.mission_name.find(' (')]
+        mission_groups[short_name] = [mission_var.mission_name for mission_var in SC2Mission
+                                      if mission_var.flags & MissionFlag.HasRaceSwap|MissionFlag.RaceSwap

```

Comment:
This condition will always trigger, as the operator precedence is `(mission_var.flags & MissionFlag.HasRaceSwap) | MissionFlag.RaceSwap`. Needs some parentheses, or can be removed entirely as the second condition is sufficient (not sure which one runs faster, but not sure it's worth worrying overmuch about optimizing a loop with only 100 or so members)

Reply:
Removed it entirely, the 2nd condition is the one that mattered